snmp/tests/common.py

```diff
@@ -299,3 +303,36 @@ def assert_common_device_metrics(
     aggregator.assert_metric(
         'snmp.devices_monitored', metric_type=aggregator.GAUGE, tags=tags, count=count, value=devices_monitored_value
     )
+
+
+def filter_tags(tags, exclude_tag_keys):
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
In API filter usually means including provided fields. It's ok here as it is tests related, I would have called it `removeTags`

Reply:
👍  Addressed here https://github.com/DataDog/integrations-core/pull/11057/commits/95c9959275079c37655df448c9876058e4e55373